lib/parallel_guest_migration_base.pm

```diff
@@ -1750,6 +1750,26 @@ sub check_peer_test_run {
     return $_peer_test_run_result;
 }
 
+=head2 filter_migration_tests
+
+Filters migration tests based on specified criteria.
+
+=cut
+
+sub filter_migration_tests {
+    my ($self, @guest_migration_test) = @_;
+    my @filtered_tests;
+
+    foreach my $migration_criteria (@guest_migration_test) {
+        if ($migration_criteria eq 'virsh_live_native_p2p_manual_postcopy') {
+            record_soft_failure('bsc#1221812 - qemu coredump when virsh migrate postcopy + virsh migrate-postcopy on sle15sp6 kvm host.');
+        } else {
+            push @filtered_tests, $migration_criteria;
```

Comment:
Just curious -- how would you update code when bsc#1221812 is fixed?

Reply:
Delete code! I think.